components/Header.tsx

```diff
@@ -6,8 +6,11 @@ import Image from "next/image";
 import Link from "next/link";
 import { AiOutlineClose } from "react-icons/ai";
 import { TbMenu } from "react-icons/tb";
+import useCreators from "@/hooks/useCreators";
+import DropdownButton from "./DropdownButton";
 
 export default function Header() {
+  const {isWhitelisted, creators } = useCreators();
```

Comment:
estas usando `creators`???
```suggestion
  const {isWhitelisted } = useCreators();
```